lab_1_keywords_tfidf/main.py

```diff
@@ -30,11 +39,16 @@ def remove_stop_words(tokens: list[str], stop_words: list[str]) -> Optional[list
 
     Returns:
     List[str]: Token sequence that does not include stop words
-
     In case of corrupt input arguments, None is returned
     """
-    pass
-
+    if not tokens:
+        return None
+    if not isinstance(tokens, list) or not isinstance(stop_words, list):
+        return None
+    for word in stop_words:
+        while word in tokens:
+            tokens.remove(word)
```

Comment:
instead of modifying an input argument, consider creating a new one and adding those words from `tokens` that are acceptable

Reply:
done